contrib/dracut/90zfs/module-setup.sh.in

```diff
@@ -39,7 +39,7 @@ install() {
 	dracut_install @sbindir@/zfs
 	dracut_install @sbindir@/zpool
 	# Include libgcc_s.so.1 to workaround zfsonlinux/zfs#4749
-	if type gcc-config 2>&1 1>/dev/null; then
+	if command -v gcc-config 2>&1 1>/dev/null; then
 		dracut_install "/usr/lib/gcc/$(s=$(gcc-config -c); echo ${s%-*}/${s##*-})/libgcc_s.so.1"
 	else
 		dracut_install /usr/lib/gcc/*/*/libgcc_s.so.1
```

Comment:
This fixes the first conditional but we also need a second part.  As mentioned in the issue

>  The symlink in /usr/lib/gcc/*/*/ is called libgcc_s.so, and points to /lib64/libgcc_s.so.1.

We can't assume `/usr/lib/gcc/*/*/libgcc_s.so.1` is correct and should add a check.